_posts/2019-06-11-sgid-then-and-now.md

```diff
@@ -0,0 +1,128 @@
+
+[![SGID Early 2000s]({{ "/images/SGID_early2000s_sm.png" | prepend: site.baseurl }}){: .flex .flex--center}]({{ "/images/SGID_early2000s.png" | prepend: site.baseurl }})
+
+### Where is the SGID Now?
+{: .text-left}
+
+Technology and the use of GIS have come a long way since the inaugural SGID of 1991.   What hasn't changed, though, is the notion of a centralized repository for searching and fetching statewide GIS data.  Sure, file formats and how we access data has changed, but the need for open, readily-accessible GIS data remains the same.
+
+The current makeup of the SGID is distributed, by design.  Internally, much of the framework data is managed on a SQL ArcSDE Database.  From there, data layers are distributed accordingly.
+
+The [sgid index]({{ "/data/sgid-index/" | prepend: site.baseurl }}) and user guide are located on [AGRC’s website]({{ "/data/" | prepend: site.baseurl }}).  Here you will find descriptions of the data, update dates, data downloads, as well as links to other places where you can access the data.  It is recommended that you begin your data search there.
+
+![SGID 2019 User Guide]({{ "/images/SGID_2019_webpage_data.png" | prepend: site.baseurl }}){: .flex .flex--center}
+
+In order to meet the diverse needs of our users and applications, SGID data is served on a variety of platforms in a variety of formats, including direct database access, downloadable data, [ArcGIS Online](https://gis.utah.gov/data), an [open data portal](https://opendata.gis.utah.gov/), and API web services.
+
+Here are some current SGID stats, showing top accessed data layers by platform:
+
+- **[AGRC Website](https://gis.utah.gov/data)** - Page-views from the past 12 months - rounded to the nearest one-hundred
+  - Parcels - (30,900 page-views)
+  - Elevation and Terrain - (20,400)
+  - Aerial Photography - (20,100)
+  - Transportation - (18,200)
+  - USGS Scanned Topo Maps - (15,900)
+  - Boundaries - Landownership - (15,600)
+  - Boundaries - City, County, and State - (15,300)
+  - Oil and Gas - (8,000)
+  - Water - (7,600)
+  - Basemap and Imagery - (5,300)
+
+- **[ArcGIS Online](https://utah.maps.arcgis.com)** - The most popular content from June 2013 to May 2019, rounded to the nearest one-thousand
+  - SBI Operations (which includes: Law Enforcement Boundaries, City Courts, County Courts, State District Courts, UHP Dispatch Zones) - (67,000 views)
+  -  Roads (minor and major roads combined stats) - (64,000)
+  -  County Boundaries - (47,000)
+  -  Municipal Boundaries - (35,000)
+  -  Parcels - (28,000)
+  -  Lakes - (26,000)
+  -  Address Points - (17,000)
+  -  Zip Codes - (17,000)
+
+- **SQL ArcSDE Database** - Desktop-user Views, from the past 12 months (approximately), rounded to the nearest one-hundred
```

Comment:
```suggestion
- **SQL ArcSDE Database** - Desktop-user views from the past 12 months (approximately) rounded to the nearest one-hundred
```

Reply:
Are "desktop-user views" different from the page views referenced for the other data layers?...